github/ci/prow/files/jobs/kubevirt/project-infra/project-infra-periodics.yaml

```diff
@@ -148,3 +148,30 @@ periodics:
     - name: token
       secret:
         secretName: oauth-token
+- name: periodic-project-infra-autoowners
+  interval: 24h
+  decorate: true
+  spec:
+    containers:
+    - image: docker.io/kubevirtci/autoowners@sha256:025f8ba96ffdc6d3adf17a0058898e17a8fe814314ec3c4bd2af9812aeeda7b7
+      env:
+      # required to fix git issue on centos:7 https://github.com/git/git/commit/92bcbb9b338dd27f0fd4245525093c4bce867f3d
+      - name: GIT_COMMITTER_NAME
+        value: kubevirt-bot
+      - name: GIT_COMMITTER_EMAIL
+        value: rmohr+kubebot@redhat.com
+      command:
+      - "/bin/sh"
+      - "-c"
+      - >
+        mkdir -p /tmp && cd /tmp &&
+        git clone https://github.com/kubevirt/project-infra.git &&
```

Comment:
A nit: you can do a shallow clone here to reduce the chance of clone failure and speed up the process.

Reply:
I've never used shallow clone I must admit. So IIUC using `--depth 1` this will i.e. just fetch the latest commit? But you still can work normally on it, right? I.e. the autoowners will create a PR from the checkout, so that will work?

Reply:
I think with the current repository size this is not an issue. Maybe we can do this when this becomes a problem?

Reply:
That's why I don't think that we should block this PR... I thought that this was on the `kubevirt/kubevirt` repo... since it's in project-infra, we're good.

Let's merge.